### PR TITLE
[admin-tool]Modified integration test for blob-transfer

### PR DESCRIPTION
**Pull request by sebas-inf**
## Summary, imperative, start upper case, don't end with a period

Modified an integration test to account for DaVinci blob transfer

## How was this PR tested?

Ran the test and it worked as expected
## Does this PR introduce any user-facing changes?

- [X] No. You can skip the rest of this section.
- [ ] Yes. Make sure to explain your proposed changes and call out the behavior change.

### CODE REVIEW

File: clients/venice-admin-tool/src/test/java/com/linkedin/venice/TestAdminTool.java [hunk 1]
```diff
@@ -74,13 +74,15 @@ public void testAdminToolRequiresSSLConfigFile() {
   public void testAdminUpdateStoreArg() throws ParseException, IOException {
```
Comment (haoxu07) [inline clients/venice-admin-tool/src/test/java/com/linkedin/venice/TestAdminTool.java:74]: Seems this is a unit test?

Comment (sebas-inf) [reply clients/venice-admin-tool/src/test/java/com/linkedin/venice/TestAdminTool.java:74]: I was told that it was an integration test

**Comment by sixpluszero** [comment]
Let's follow the convention of the title tag to [admin-tool] as it is only touching admin tool

File: clients/venice-admin-tool/src/test/java/com/linkedin/venice/TestAdminTool.java (hunk 1 above)
Comment (sixpluszero) [reply clients/venice-admin-tool/src/test/java/com/linkedin/venice/TestAdminTool.java:74]: Yeah I did not checked how it was implemented before I told you to add/modify an integration test, but the integration test usually spin up the whole venice cluster in the test suite and run end to end. This only creates admin tool object so it is not.

Comment (sebas-inf) [reply clients/venice-admin-tool/src/test/java/com/linkedin/venice/TestAdminTool.java:74]: oh ok

**Review by sixpluszero — APPROVED** [review]
Could you update the title to update "integration test" to "unit test" before you merge the PR? Thank you!

**Pull request merged**